File: android/src/main/java/com/seewo/flutter/gt3_captcha/Gt3CaptchaPlugin.java
package com.seewo.flutter.gt3_captcha;

import androidx.annotation.NonNull;

import io.flutter.embedding.engine.plugins.FlutterPlugin;
import io.flutter.plugin.common.BinaryMessenger;

/** Gt3CaptchaPlugin */
public class Gt3CaptchaPlugin implements FlutterPlugin {

  @Override
  public void onAttachedToEngine(@NonNull FlutterPluginBinding flutterPluginBinding) {
    BinaryMessenger messenger = flutterPluginBinding.getBinaryMessenger();
    flutterPluginBinding
            .getPlatformViewRegistry()
            .registerViewFactory(
                    "plugins.flutter.io/gt3_captcha_view",
                    new Gt3CaptchaViewFactory(messenger, /*containerView=*/ null));
  }

  @Override
  public void onDetachedFromEngine(@NonNull FlutterPluginBinding binding) {

  }
}

File: ios/Classes/Gt3CaptchaPlugin.m
#import "Gt3CaptchaPlugin.h"
#import "Gt3CaptchaViewFactory.h"

@implementation Gt3CaptchaPlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
    Gt3CaptchaViewFactory *gt3CaptchaViewFactory = [[Gt3CaptchaViewFactory alloc] initWithMessenger:registrar.messenger];
    [registrar registerViewFactory:gt3CaptchaViewFactory withId:@"plugins.flutter.io/gt3_captcha_view"];
}

@end
